umm-spec-lib/resources/xml-schemas/echo10/Collection.xsd

```diff
@@ -271,7 +291,7 @@ xmlns:xs="http://www.w3.org/2001/XMLSchema">
               <xs:annotation>
                 <xs:documentation> Any EOS and non-EOS data and data
                 products, that are not SCIENCE_QUALITY and do not
-                fall under NEAR_REAL_TIME holdings.</xs:documentation>
+                fall under NEAR_REAL_TIME, LOW_LATENCY and EXPEDITED holdings.</xs:documentation>
```

Comment:
On line 292 the documentation for OTHER can you replace the text with the following: 
The data is ready for end user consumption between the EXPEDITED and SCIENCE_QUALITY acquisition timeframes. This data is not fully processed and is not retained by EOSDIS once the SCIENCE_QUALITY product is archived.